tests/unit/core/config/test_user_config.py

```diff
@@ -261,6 +261,32 @@ def test_sca_config(self, local_config_path):
             until=None,
         )
 
+    def test_sca_ignore_vuln_until(self, local_config_path):
+        """
+        GIVEN a local config file with sca configs
+        WHEN setting an ignore vulnerability param with a past datetime
+        THEN it's ignored
+        """
+        write_yaml(
+            local_config_path,
+            {
+                "version": 2,
+                "sca": {
+                    "ignored_vulnerabilities": [
+                        {
+                            "identifier": "GHSA-aaaa-bbbb-cccc",
+                            "path": "Pipfile",
+                            "comment": "Not my prob",
+                            "until": "2023-05-01T00:00:00.000Z",
```

Comment:
What happens if you have no "Z" in the config ? (ie: you have a naive datetime)

Reply:
Modify code to transform all datetimes' timezone to UTC